net.granjow.slowmovideo.yaml

```
@@ -0,0 +1,75 @@
+app-id: net.granjow.slowmovideo
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+
+command: slowmoUI
+rename-icon: AppIcon
+rename-desktop-file: slowmoUI.desktop
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  # The app creates a directory which does not work through the portal
+  - --filesystem=xdg-videos
+
+
+modules:
+
+  - name: opencv
+    # Taken from https://github.com/flathub/org.d3cod3.Mosaic/blob/be9aea900361940120341b2ad6fe7d97bec39c66/org.d3cod3.Mosaic.yml#L152
+    buildsystem: cmake-ninja
+    builddir: true
+    cleanup:
+        - '/include/'
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DOPENCV_GENERATE_PKGCONFIG=ON
+      #- -OPENCV_ENABLE_NONFREE=ON
+      - -DWITH_OPENCL=ON
+      - -DWITH_OPENGL=ON
+      - -DWITH_TBB=ON
+      - -DWITH_VULKAN=ON
+      - -DBUILD_WITH_DEBUG_INFO=OFF
+      - -DBUILD_TESTS=OFF
+      - -DBUILD_PERF_TESTS=OFF
+      - -DBUILD_EXAMPLES=OFF
+      - -DINSTALL_C_EXAMPLES=OFF
+      - -DINSTALL_PYTHON_EXAMPLES=OFF
+      # - -DOPENCV_EXTRA_MODULES_PATH="contrib/modules"
+      - -DWITH_1394=OFF
+      - -DWITH_PROTOBUF=ON
+      # - -DOPENCV_EXTRA_MODULES_PATH=../opencv_contrib-master/modules ../opencv-master
+      #- -DBUILD_LIST=calib3d,core,dnn,features2d,flann,highgui,imgcodecs,imgproc,java_bindings_generator,ml,objdetect,photo,python_bindings_generator,python_tests,stitching,video,videoio
+    sources:
+        - type: archive
+          url: https://github.com/opencv/opencv/archive/3.4.18.tar.gz
+          sha256: d6a69aa4a775a074662685b9bc2b07b88fb867f19d939b03ad281571b62f2620
+
+  - ffmpeg.yaml
+
+  - name: slomoVideo
+    buildsystem: cmake-ninja
+    builddir: true
+    post-install:
+        - install -D -ma+r /app/share/icons/AppIcon.png /app/share/icons/hicolor/256x256/apps/AppIcon.png
+        - install -D -ma+r net.granjow.slowmovideo.appdata.xml /app/share/appdata/net.granjow.slowmovideo.appdata.xml
```

```suggestion
        - install -D -m644 ../net.granjow.slowmovideo.appdata.xml /app/share/appdata/net.granjow.slowmovideo.appdata.xml
```

You said `builddir: true`, post-install is done from it.